datasets/dk/pep/dk_pep.yaml

```diff
@@ -66,6 +66,7 @@ lookups:
           - Stillingsbetegnelse
           - roh
           - Stilling
+          - Sonne-Schmidt
```

Comment:
are you sure this a column heading and not a surname?

Reply:
probably a mistake on their side. it was there a couple of hours ago but it seems that they've already fixed it
![Screenshot 2024-12-04 at 15 58 47](https://github.com/user-attachments/assets/d5a4c1f6-9d59-4a68-b80b-2e88c125fc82)